### PR TITLE
cockpit-ws: add page

### PR DESCRIPTION
**Pull request by rajatmohan22**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Thanks for your contribution. There is another PR regarding this #9145. I prefer your version as there are more commands. I will review your version of this page, I will request the other contributor to remove this page. If you have suggestions for other commands feel free to drop it in #9145.

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:3]:
```suggestion
> Used for communication between the browser application and various configuration tools and services like `cockpit-bridge`.
```

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:6]:
```suggestion
- Check which certificate `cockpit-ws` will use:
```
I am not sure if this command is fine as it isn't a part of `cockpit-ws` we don't usually add external commands to these pages. @navarroaxel What do you think?

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:10]:
```suggestion
-Display help:
```
This is what we use for help commands across all pages (except some). **Also, move the help command to the last in this page**

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:14]:
```suggestion
- Authenticate via SSH at `127.0.0.1` port `22`:
```
Adding this suggestion to better align with official documentation.

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:18]:
```suggestion
- Serve HTTP requests to a specific PORT instead of port `9090`:
```

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:20]:
```suggestion
`cockpit-ws --port {{PORT}}`
```
We enclose command arguments and placeholders in `{{}}`.

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:22]:
```suggestion
- Bind to a specific ADDRESS instead of binding to all available addresses:
```

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:24]:
```suggestion
`cockpit-ws --address {{ADDRESS}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [reply pages/linux/cockpit-ws.md:24]: > Sure, I'll make these changes and push again immediately. Could you just confirm about the certificate command?

You can implement other suggestions. Let's wait for insights regarding that command alone.

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:6]:
```suggestion
- Check which certificate `cockpit-ws` will use:
```

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:3]:
```suggestion
> Communicate between the browser application and various configuration tools and services like `cockpit-bridge`.
```

**Comment by rajatmohan22** [comment]
@kbdharun Done!

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [reply pages/linux/cockpit-ws.md]: Exactly, this correspond to new page called `remotectl`

**Comment by rajatmohan22** [comment]
Should i remove the certification command and push again ?

File: pages/linux/cockpit-ws.md
Comment (rajatmohan22) [reply pages/linux/cockpit-ws.md]: Should I remove it and push again?

Comment (rajatmohan22) [reply pages/linux/cockpit-ws.md]: I am new to Open Sourcing and don't know how these "request reviews" work. Pardon me if Ive done anything wrong.
Like mentioned I have removed the remotectl command and pushed again.

**Comment by kbdharun** [comment]
@rajatmohan22 Can you remove the other 2 pages? If they are new additions create a new branch from tldr: main in your fork and create a PR from it.

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:6]:
```suggestion
- Authenticate via SSH at `127.0.0.1` port `22`:
```

**Comment by rajatmohan22** [comment]
> @rajatmohan22 Can you remove the other 2 pages? If they are new additions create a new branch from tldr: main in your fork and create a PR from it.

Yes I have done it, Just verify once

File: pages/linux/cockpit-ws.md
Comment (kbdharun) [reply pages/linux/cockpit-ws.md:6]: @rajatmohan22 The page LGTM, Can you commit this suggestion alone?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by rajatmohan22** [comment]
@navarroaxel Hey! do you mind reviewing the PR once? If its all right, please do approve it

**Comment by rajatmohan22** [comment]
@kbdharun Hey, could you add more people for review if possible please ?

**Comment by rajatmohan22** [comment]
@navarroaxel Hello, could you please check this PR and approve it if all is good ?

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [inline pages/linux/cockpit-ws.md:10]:
```suggestion
- Start an HTTP server on a specific port:
```

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [inline pages/linux/cockpit-ws.md:12]: We use `snake_case` for placeholders
```suggestion
`cockpit-ws --port {{port}}`
```

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [inline pages/linux/cockpit-ws.md:14]:
```suggestion
- Start and bind to a specific IP address (defaults to `0.0.0.0`):
```

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [inline pages/linux/cockpit-ws.md:16]:
```suggestion
`cockpit-ws --address {{ip_address}}`
```

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [inline pages/linux/cockpit-ws.md:18]:
```suggestion
- Start without TLS:
```

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [inline pages/linux/cockpit-ws.md:6]: Perhaps this better
```suggestion
- Start with authentication via SSH at `127.0.0.1` port `22` enabled:
```

**Review by kbdharun — APPROVED** [review]

**Comment by rajatmohan22** [comment]
@navarroaxel Would you mind reviewing the changes now please?

File: pages/linux/cockpit-ws.md
Comment (navarroaxel) [inline pages/linux/cockpit-ws.md:6]: A connector is missing here
```suggestion
- Start with authentication via SSH at `127.0.0.1` with port `22` enabled:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
